account_mtd_vat/models/mtd_vat_calculation.py

```diff
@@ -111,16 +111,22 @@ def calculate_originator_taxcode_balance(self, tax_code_record, date_from, date_
             ('date', '>=', date_vat_period),
             ('date', '<=', date_to),
             ('vat', '=', vat_posted),
-            ('tax_line_id.id', '=', tax_code_record.id)])
+            ('parent_state', '=', 'posted'),
+            ('tax_line_id', '=', tax_code_record.id)])
 
         line_ids = []
         credit = []
```

Comment:
just noticed this, I know it's not related to this PR but we can do it as a drive-by fix.

Can we change `credit` to `credits`, and `debit` to `debits` please :)

Reply:
Ok